targets/ipq806x

```diff
@@ -1,7 +1,29 @@
-ATH10K_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+# The QCA9980 is discontinued from Qualcomm. It didn't receive any firmware-update since >4 years.
+# Any kind of meshing was never implemented for this chip. It is (most likely) broken forever.
+# The QCA9984 on the other hand works fine for 11s meshes on both bands.
 
+QCA9980_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+QCA9984_PACKAGES='kmod-ath10k -kmod-ath10k-ct ath10k-firmware-qca9984 -ath10k-firmware-qca9984-ct'
 
-# TP-Link
 
+#
```

Comment:
If it is broken indefinately we can either decide to drop and not accept QCA9980 radios completely, or use this file to remind us of tried-and-failed devices, in which case I would like an if broken block surrounding all QCA9980 devices.

Yes I know the target is currently marked as BROKEN, but that makes unbreaking it easier at a later point, when everything is already reasonably documented.

Reply:
Why don't you use the Candelatech firmware? It supports QCA99X0. Can you please explain why you select the outdated QCA firmware at all?

Reply:
Okay I read the thread about ipq806x. The ct-driver was updated some months ago with changes to QCA99X0. Can you please test the router with ct-firmware only? Don't select both firmwares. And if there are issues, please fill a bug report at CT. This driver is being actively developed.

Reply:
@CodeFetch I do not own a QCA99x0 based device. They can be considered the AR1A of Wave 2 chips so unless someone funds me i won't invest any time there. We weren't able to bring up any mesh/AP combination up at all with any firmware/driver combination some months back.

For the QCA99x4: The -ct firmware crashed for me multiple times within 24 hours so i will stick with the one from QCA here. Same goes for the driver. I see no reason to switch to the -ct one when the one from QCA works flawlessly for our scenario.

Reply:
Thank you for the hint with the crashing. I assume you would get SWBA overrun messages, too. Then it is a known bug.